lib/collection/ui/Game.tsx

```diff
@@ -253,7 +264,7 @@ export function Game() {
   } as const;
 
   const gameState: GameState = (() => {
-    // return gameStates.testing;
+    return gameStates.testing;
```

Comment:
We don't want a testing override on main
```suggestion
    // return gameStates.testing;
```